app/scripts/modules/core/src/artifact/react/ExpectedArtifactSelector.spec.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import { ExpectedArtifactService, IExpectedArtifact } from 'core';
+import { ExpectedArtifactSelector } from './ExpectedArtifactSelector';
+
+const artifact = (kind: string, type: string): IExpectedArtifact => {
+  const ea = ExpectedArtifactService.createEmptyArtifact(kind);
+  ea.matchArtifact.type = type;
+  return ea;
+};
+
+fdescribe('<ExpectedArtifactSelector/>', () => {
```

Comment:
`fdescribe` I think causes only this test to run (and all others to be ignored)

Reply:
Doh. _Artifact_ of development.